practices/code-review.md

```diff
@@ -60,11 +60,30 @@ Here's the general process that we use:
 
 ## What to look out for
 
-* Are there syntactic inconsistencies the linter did not catch?
-* Overly complex code.
-* PRs which are non-atomic and should be split into separate PRs, e.g. a new UI feature combined with an unrelated configuration change.
-* Code which could be split up into reusable modules.
-* Non-performant code. Suggest testing with [WebPageTest](https://www.webpagetest.org/) and/or other tools.
-* Inaccessible code. Suggest testing with [pa11y](https://github.com/pa11y) and/or other tools.
+* **Accessibility**
+
+  * Accessibility problems. Suggest testing with [pa11y](https://github.com/pa11y) and/or other tools.
     * [internal-only pa11y dashboard](http://pa11y.springernature.com/)
-* Do you see something which may have wider ramifications than the author anticipated? Double check they are aware which can of worms they have just opened :)
+
+* **Complexity**
+
+  * Overly complex code: code that solves problems we don't have, or makes things more complex instead of simplifying things.
```

Comment:
"... makes things more complex instead of less."

(removes vague 'things', shortens sentence 😉)

Reply:
I know all this stuff was in the original, but hey ho ☺️

Reply:
Actually I added that bit, so good point :+1: